exploratory_data_analysis/eda_gabi.py

```diff
@@ -0,0 +1,224 @@
+import pandas as pd
+from exploratory_data_analysis import config
+from utils.plotting import plot_lines_by
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+# OIL Consumption Dataset
+df_oil_consum_pc = pd.read_csv(config.oil_consum_pc)
+df_oil_consum_pc.head()
+df_oil_consum_pc.info()
+
```

Comment:
are you printing that in the console? 

if that's the eda_gabi, leave it in your branch. Let's keep in "main" only the files that will be used in producing the model.

Reply:
Yes I do. I have had just commit and push my things to the repo. I guess this was included in the PR as this PR is open since 1 week

Reply:
Leave it in your branch. Do not merge it to the main project until is not "official".